src/deploy/zoneless.rs

```diff
@@ -0,0 +1,69 @@
+use crate::commands::subdomain::Subdomain;
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::toml::RouteConfig;
+use crate::terminal::message::{Message, StdOut};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct ZonelessTarget {
+    pub account_id: String,
+    pub script_name: String,
+}
+
+impl ZonelessTarget {
+    pub fn build(script_name: &str, route_config: &RouteConfig) -> Result<Self, failure::Error> {
+        match route_config.account_id.as_ref() {
+            // TODO: Deserialize empty strings to None; cannot do this for account id
```

Comment:
good callout. we should make an issue referencing this line.

Reply:
This was an existing comment, so an issue may already exist.

Reply:
It may also not be a relevant comment anymore? Most strings use a `string_empty_as_none` deserializer now so it could just be outdated.

Reply:
i'll try to get through inline TODOs at some stage